Documentation/PermissionsManagement/GeneralRecommendations/Index.rst

```diff
@@ -0,0 +1,145 @@
+a security risk. Every so often, it's good to check and see which accounts
+are still needed and which can be removed.
+
+Each user account should be protected with a strong password to minimize the risk
+of a security breach. Always adhere to :ref:`secure password selection guidelines <t3coreapi:security-secure-passwords>`
+when creating a password for a new user or updating a password for an existing user.
+By setting a strong example and informing users about your security policies,
+you reinforce the importance of cybersecurity. It is also strongly advised
+to set up Multi-factor authentication (MFA) which will provide an additional
+layer of security.
+
+.. note::
+    - Grant users only the access they truly need
+    - Instead of using administrative accounts, create groups for specific roles, such as editor, and assign these roles to users
+    - Regularly maintain backend user accounts, removing any that are no longer needed
+    - Always establish secure passwords for users
+
+.. _permissions-via-groups:
+
+Set permissions via groups, not user records
+============================================
+
+Some permissions or access rights can be directly configured through the backend
+user record. This allows specifying which modules the user can see, the languages
+they can use, and assigning database, file, or category mounts, as well as defining
+allowed file and folder operations. While it may be tempting to set these permissions
+directly through the user record, especially in small projects with only one or two
+accounts managing content and configuration, this approach can become problematic
+over time. It may be difficult to determine the origin of some permissions
+if they are distributed across backend users and groups. Managing all permissions
+exclusively through backend groups simplifies maintenance significantly.
+
+.. figure:: ../../Images/ManualScreenshots/PermissionsManagement/PermissionsManagementDoNotSetPermissionsOnUserRecord.png
+   :alt: User record without permissions
+
+   Avoid setting permissions directly through the backend user record
+
+Consider a scenario where several backend users have specific permissions set,
+alongside multiple groups that extend these permissions. If you need to update
+access rights by adding or removing them, and these rights are configured at both
+the user and group levels, you would have to individually edit each user account
+to ensure the permissions are correctly granted or revoked. However,
+if permissions were set exclusively for selected groups, you would only need
+to update these groups, and all users within them would automatically inherit
+the changes.
+
+.. note::
+    Configure permissions only through the backend user groups. Don’t set them on the user record.
+
+.. _file-mounts-and-files-management:
+
+File mounts and files management
+================================
+
+In the previous section, we described naming conventions for backend groups,
```

Comment:
Do not refer to previous section, you are not writing a book, ppl jump on the web ;-)